### ISSUE TITLE
[Salariés] Ajouter les conventions collectives demandées par les usagers

### ISSUE THREAD

**Issue by glopezrios** [betagouv/mon-entreprise#3020]
Jusqu'à présent, les conventions collectives non-intégrées dans le simulateur pour salariés et qui sont demandées par les usagers sont regroupées dans la rubrique des discussions "Convention collective":
https://github.com/betagouv/mon-entreprise/discussions/categories/convention-collective

Suite à la nouvelle procédure décidée en équipe, je crée donc une issue qui reprend ces éléments.

Mais pour suivre pleinement la nouvelle procédure, il faudrait créer une issue pour chaque convention à rajouter (donc pour chaque discussion). Cela paraitrait cohérent avec la logique "une issue = un problème" mais modifier toutes ces conversations en issue est un peu rébarbatif...

Donc sauf demande contraire, je laisse pour l'instant en une seule issue, sachant que cela représente probablement un gros travail d'analyser ces différentes conventions collectives puis d'intégrer leur impact sur le simulateur, par rapport au droit commun.
```[tasklist]
### Tasks
- [ ] https://github.com/betagouv/mon-entreprise/issues/3043
```

**Comment by JalilArfaoui** [comment]
Oui, en effet, vu le travail différent pour chacune, il faudra de toute façon créer une issue pour chaque CC que l'on entreprend de prendre en compte…

**Comment by VeroniqueR75** [comment]
On créera une Issue pour chaque convention collective à ajouter lorsque le besoin remontera par les usagers
